cmd/root.go

```diff
@@ -131,6 +131,8 @@ func init() {
 		baseLocalDir = os.Getenv("APPDATA")
 	case "linux":
 		baseLocalDir = filepath.Join(os.Getenv("HOME"), ".local", "share")
+	case "darwin":
+		baseLocalDir = filepath.Join(os.Getenv("HOME"), ".ficsit-cli")	
```

Comment:
The standard for this seems to be `~/Library/Application Support`, so we should use that instead. Also `baseLocalDir` is the path without the app name, that's added below.

Reply:
done